core/src/main/java/org/geysermc/geyser/command/GeyserCommandManager.java

```diff
@@ -107,7 +107,8 @@ public void register(@NonNull Command command) {
 
         // Register help commands for all extensions with commands
         for (Map.Entry<Extension, Map<String, Command>> entry : this.extensionCommands.entrySet()) {
-            registerExtensionCommand(entry.getKey(), new HelpCommand(this.geyser, "help", "geyser.commands.exthelp.desc", "geyser.command.exthelp", entry.getKey().description().id(), entry.getValue()));
+            String id = entry.getKey().description().id();
+            registerExtensionCommand(entry.getKey(), new HelpCommand(this.geyser, "help", "geyser.commands.exthelp.desc", "geyser.command.exthelp." + id, id, entry.getValue()));
```

Comment:
Just something to note, currently extensions cant register language strings so `geyser.command.exthelp.xxxxx` will never resolve.

Reply:
Not sure what language string there is to resolve, it's the permission that's set
The description string is always the same & set before the permission